src/frontend/components/accordion/accordion.ts

```diff
@@ -1,11 +1,10 @@
 import {Component, Input} from '@angular/core';
-import {NgClass} from '@angular/common';
 
 @Component({
   selector: 'accordion',
   template: require('./accordion.html'),
 })
-export default class Accordion {
+export class Accordion {
```

Comment:
This also makes it consistent with all of our other `@Component`-decorated modules, so that the `@NgModule.declarations` imports look consistent.